common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1130,33 +1131,33 @@
     "figura.docs.model_part.is_child_of": "Checks if this part is a child of the given part",
-    "figura.docs.model_part.get_pos": "Gets the position of the model part, as an offset from its position in blockbench\nOnly changes from {0,0,0} when you call setPos()",
-    "figura.docs.model_part.set_pos": "Sets the position offset for this part from its blockbench position\nNil values for position are assumed to be 0",
+    "figura.docs.model_part.get_pos": "Gets the position of the model part, as an offset from its position in BlockBench\nOnly changes from {0,0,0} when you call setPos()",
+    "figura.docs.model_part.set_pos": "Sets the position offset for this part from its BlockBench position\nNil values for position are assumed to be 0",
     "figura.docs.model_part.get_anim_pos": "Gets the position offset provided by the currently active animation of this model part",
     "figura.docs.model_part.get_true_pos": "Gets the true position of this model part, which is a sum of the position and the animation position",
-    "figura.docs.model_part.get_rot": "Gets the rotation of the model part, including its rotation in blockbench\nFor relative rotation values, check out the \"offset\" rot functions",
+    "figura.docs.model_part.get_rot": "Gets the rotation of the model part, including its rotation in BlockBench\nFor relative rotation values, check out the \"offset\" rot functions",
     "figura.docs.model_part.set_rot": "Sets the absolute rotation for this part\nNil values for rotation are assumed to be 0\nAngles are given in degrees\nFor relative rotation values, check out the \"offset\" rot functions",
-    "figura.docs.model_part.get_offset_rot": "Gets the offset rotation of the model part, offset from its rotation in blockbench\nFor absolute rotation values, check out the non-offset rot functions",
+    "figura.docs.model_part.get_offset_rot": "Gets the offset rotation of the model part, offset from its rotation in BlockBench\nFor absolute rotation values, check out the non-offset rot functions",
     "figura.docs.model_part.set_offset_rot": "Sets the offset rotation for this part\nNil values for rotation are assumed to be 0\nAngles are given in degrees\nFor absolute rotation values, check out the non-offset rot functions",
     "figura.docs.model_part.get_anim_rot": "Gets the rotation offset provided by the currently active animation of this model part",
     "figura.docs.model_part.get_true_rot": "Gets the true rotation of this model part, which is a sum of the rotation, the offset rotation and the animation position",
-    "figura.docs.model_part.get_scale": "Gets the scale of the model part, as a multiple of its blockbench size\nOnly changes from {1,1,1} when you call setScale()",
+    "figura.docs.model_part.get_scale": "Gets the scale of the model part, as a multiple of its BlockBench size\nOnly changes from {1,1,1} when you call setScale()",
```

You wouldn't believe it.
```suggestion
    "figura.docs.model_part.get_scale": "Gets the scale of the model part, as a multiple of its Blockbench size\nOnly changes from {1,1,1} when you call setScale()",
```